inlang/source-code/message-lint-rules/camelCaseId/README.md

```diff
@@ -0,0 +1,23 @@
+# What does this rule do?
+
+Checks for messages to have a snake case message id.
+
+Having the message id's in snake case format is beneficial because it allows for easier integration with other tools and libraries, like `@inlang/paraglide-js`.
```

Comment:
I think you have copied the `README.md` from the snake case ID. This file has to rename snake case to camel case.

Reply:
updated ✅